packages/myst-cli/src/process/site.ts

```diff
@@ -107,20 +125,6 @@ export async function addProjectReferencesToObjectsInv(
   return inv;
 }
 
-export async function loadProject(
-  session: ISession,
-  projectPath: string,
-  opts?: { writeToc?: boolean; reloadProject?: boolean },
-) {
-  const project = await loadProjectFromDisk(session, projectPath, {
-    warnOnNoConfig: true,
-    ...opts,
-  });
-  // Load the citations first, or else they are loaded in each call below
-  const pages = filterPages(project);
-  return { project, pages };
-}
-
```

Comment:
This function is very "thin", so I decided to remove it for readability. Can restore if needed!